core/src/components/datetime/datetime.tsx

```diff
@@ -724,7 +733,8 @@ export class Datetime implements ComponentInterface {
          */
         if (mode === 'ios') {
           const ratio = ev.intersectionRatio;
-          const shouldDisable = Math.abs(ratio - 0.7) <= 0.1;
+          // `maxTouchPoints` will be 1 in device preview, but > 1 on device
+          const shouldDisable = Math.abs(ratio - 0.7) <= 0.1 && navigator.maxTouchPoints > 1;
```

Comment:
It will no longer lock-up on browser, but it will animate early (at the 70% threshold mark). Let me see if adding the `maxTouchPoints` check when determining the IO threshold works both on device & web browser.

Reply:
Confirmed on device and web browser that using this check for assigning the threshold values on the intersection observers looks good.